src/frontend/sider/Fagsak/Personlinje/Behandlingsmeny/AInntekt/AInntekt.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import { Alert, Dropdown, Loader } from '@navikt/ds-react';
+
+import { useHentAInntektUrl } from '../../../../../hooks/useHentAInntektUrl';
+import type { IMinimalFagsak } from '../../../../../typer/fagsak';
+
+interface IProps {
```

Comment:
Kan trygt droppe `I` prefixen 😄 

```suggestion
interface Props {
```

Reply:
Enig. Tror jeg kopierte props fra et annet sted, men glemte å fjerne `I`